README.md

```diff
@@ -1,8 +1,21 @@
 OkHttp
 ======
 
-An HTTP & HTTP/2 client for Android and Java applications. For more information see [the
-website][website] and [the wiki][wiki].
+An HTTP & HTTP/2 client for Android and Java applications.
+
+Docs
+----
+
+ * [**User's Guide**](CALLS.md) (includes [Calls](CALLS.md), [Connections](CONNECTIONS.md),
```

Comment:
Why does users guide link to CALLS.md, does final render have menu etc?

Reply:
You got it.